node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/DnsNameResolver.java

```diff
@@ -39,6 +45,18 @@
         }
     }
 
+    @Override
+    public Optional<String> getHostname(String ipAddress) {
+        try {
+            String hostname = InetAddress.getByName(ipAddress).getHostName();
```

Comment:
This will consult `/etc/hosts` and thus does not match the javadoc for this class. However, if you were to use `lookupName` you would need to convert the IP address to the pointer domain name (e.g. "rev-ip.in-addr.arpa"), but I think this is fine for now.